R/runCCA.R

```diff
@@ -180,19 +181,27 @@ setGeneric("addRDA", signature = c("x"),
     #
     x <- as.matrix(t(x))
     variables <- as.data.frame(variables)
+    
+    if( any(is.na(variables)) && isTRUE(all.equal(na.action, na.fail)) ){
+      stop("Variables contain missing values. Set na.action to na.exclude",
+           " to remove samples with missing values.", call. = FALSE)
+    }
```

Comment:
SHould this message be something else? That is because the option can be also na.omit. We do not want to make the decision for user.